django/db/models/options.py

```diff
@@ -667,6 +668,28 @@ def get_ancestor_link(self, ancestor):
                 # links
                 return self.parents[parent] or parent_link
 
+    def get_path_to_parent(self, parent):
+        """
+        Get path from current model to a parent model.
```

Comment:
Suggested wording: "Return a list of PathInfos containing the path from the current model to a parent model, or an empty list if...."

Could you also describe the circumstances where the list will be length > 1?